docs/swagger/gncs-api.yml

```diff
@@ -174,13 +247,17 @@ components:
         expires_at:
           type: string
           format: date-time
+        created_at:
+          type: string
+        updated_at:
```

Comment:
bin mir nicht sicher wie das funktioniert. kann das token geupdated werden?

Reply:
ne eigentlich nicht, dann macht updated_at kein sinn. müsste man überlegen ob created_at auch weg kann, da expires_at ja reichen müsste

Reply:
Ich würd created_at drin lassen. Und wenn es nur dazu da ist, die expiration zu testen

Reply:
lol, die Doku is falsch. token hat gar kein updatedAt